lib/jsdom/living/helpers/selectors.js

```diff
@@ -1,47 +1,13 @@
 "use strict";
 
-const nwsapi = require("nwsapi");
+const domSelector = require("@asamuzakjp/dom-selector");
 
-const idlUtils = require("../generated/utils");
-
-function initNwsapi(node) {
-  const { _globalObject, _ownerDocument } = node;
-
-  return nwsapi({
-    document: idlUtils.wrapperForImpl(_ownerDocument),
-    DOMException: _globalObject.DOMException
-  });
-}
-
-exports.matchesDontThrow = (elImpl, selector) => {
-  const document = elImpl._ownerDocument;
-
-  if (!document._nwsapiDontThrow) {
-    document._nwsapiDontThrow = initNwsapi(elImpl);
-    document._nwsapiDontThrow.configure({
-      LOGERRORS: false,
-      VERBOSITY: false,
-      IDS_DUPES: true,
-      MIXEDCASE: true
-    });
-  }
-
-  return document._nwsapiDontThrow.match(selector, idlUtils.wrapperForImpl(elImpl));
-};
-
-// nwsapi gets `document.documentElement` at creation-time, so we have to initialize lazily, since in the initial
-// stages of Document initialization, there is no documentElement present yet.
-exports.addNwsapi = parentNode => {
+exports.addDomSelector = parentNode => {
```

Comment:
It looks like this is not needed at all anymore since it doesn't perform any initialization. It's just a global singleton library. In which case this file can be removed and `require("@asamuzakjp/dom-selector")` can be used directly.

Reply:
Removed.